Oblig 2/Oppgave 2.ipynb

```diff
@@ -18,18 +18,31 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    	search = HalvingGridSearchCV( #HalvingGridSearchCV is significanly faster than GridSearchCV while giving just as accurate results, but it is experimental...
</pre><p>Spennende metode.. Har aldri sett den før</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Bsingstad/DMBA2024/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>